RunningOrder/Shared/Components/Search/SearchBarSuggestions.swift

```diff
@@ -30,29 +36,38 @@ extension SearchBarSuggestions {
 
         var body: some View {
             ScrollView {
+//                Picker("Search Scope", selection: $logic.searchScope) {
+//                    ForEach(Scope.allCases, id: \.self) { scope in
+//                        Text(scope.title)
+//                            .tag(scope)
+//                    }
+//                }
+//                .labelsHidden()
+//                .pickerStyle(SegmentedPickerStyle())
+//                .padding()
+
                 VStack(alignment: .leading) {
-                    if !logic.filteredStories.isEmpty {
-                        ForEach(logic.filteredSearchSections, id: \.id) { section in
-                            Section(header: Text(section.type.title).font(.headline).bold()) {
-                                Divider()
-
-                                ForEach(Array(section.items)) { item in
-                                    SuggestionRow(imageName: item.icon, suggestion: item.name)
-                                        .onTapGesture { searchManager.selectItem(item) }
-                                }
+                    ForEach(logic.filteredSearchSections) { section in
+                        Section(header: header(section.type.title)) {
+                            Divider()
+
+                            ForEach(Array(section.items)) { item in
+                                SuggestionRow(imageName: section.type.icon, suggestion: item.name)
+                                    .onTapGesture { searchManager.selectItem(item) }
                             }
                         }
-                    } else {
-                        Spacer()
-                        Label("No matching stories, epics found", systemImage: "magnifyingglass")
-                            .padding()
-                        Spacer()
```

Comment:
why don't you handle this case anymore?

Reply:
because there is alway at least one section : the filter one